src/ModelBinding/ValidationExtensions.cs

```diff
@@ -6,6 +6,11 @@ namespace Botwin.ModelBinding
 
     public static class ValidationExtensions
     {
+        /// <summary>
+        /// Retrieve formatted validation errors
+        /// </summary>
+        /// <param name="result"><see cref="ValidationResult"/></param>
+        /// <returns>Property names and associated error messages <see cref="IEnumerable{dynamic}"/></returns>
```

Comment:
Maybe `<see cref="IEnumerable{dynamic}"/> of property names and associated error messages`